win32ss/user/winsrv/consrv/condrv/text.c

```diff
@@ -1144,7 +1144,8 @@ ConDrvFillConsoleOutput(IN PCONSOLE Console,
                 break;
 
             case CODE_ATTRIBUTE:
-                Ptr->Attributes = Code.Attribute;
+                Ptr->Attributes &= ~0xFF;
```

Comment:
So if there are some leading/trailing character attributes there (because the corresponding characters have these characteristics), these attributes get removed?

Reply:
The leading/trailing attributes will be remained.
The `FOREGROUND_* and BACKGROUND_*` attributes will be exchanged.

Reply:
magic number?

Reply:
oh yes you're right. Regarding 0xFF you can keep it I think (I don't have any problems about that, we understand it's only the low byte that gets filtered out).